src/fondue/Fondue.js

```diff
@@ -221,19 +226,29 @@ export default class Fondue {
 		return fvar;
 	}
 
+	// Return all color format tables.
+	// Usage:
+	//   fondue.colorFormats -> return e.g. ["SVG "] or empty array
+	get colorFormats() {
+		// const tables = Object.keys(this._raw()); // Why doesn't this work??
```

Comment:
What's not working exactly? What does `this._raw()` return?

Reply:
An empty array, while in `get tables()` it returns a list of all the tables. I don't get why it behaves differently here?

Reply:
Spooky! Could it behave differently because of the font type? `colorFormats()` I suppose is mostly used for color fonts, whereas `get tables()` is used for all fonts.